linode/acceptance/test_util.go

```diff
@@ -260,6 +262,51 @@ func CheckResourceAttrNotEqual(resName string, path, notValue string) resource.T
 	}
 }
 
+func CheckResourceAttrListContains(resName, path, desiredValue string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[resName]
+		if !ok {
+			return fmt.Errorf("Not found: %s", resName)
+		}
+
+		length, err := strconv.Atoi(rs.Primary.Attributes[path+".#"])
+		if err != nil {
+			return fmt.Errorf("attribute %s does not exist", path)
+		}
+
+		for i := 0; i < length; i++ {
+			if rs.Primary.Attributes[path+"."+strconv.Itoa(i)] == desiredValue {
+				return nil
+			}
+		}
+
+		return fmt.Errorf("Desired value not found in resource attribute")
+	}
+}
+
+func LoopThroughStringList(resName, path string, listValidateFunc ListAttrValidateFunc) resource.TestCheckFunc {
```

Comment:
I was thinking about this naming a bit, maybe we could make it something like `ValidateResourceAttrList`?